### PR TITLE
Add `wait` parameter to prediction creation methods

### PR DESCRIPTION
**Pull request by mattt**

### CODE REVIEW

File: replicate/prediction.py [hunk 1]
```diff
@@ -603,6 +610,18 @@ async def async_cancel(self, id: str) -> Prediction:
         return _json_to_prediction(self._client, resp.json())
 
 
+def _create_prediction_headers(
+    *,
+    block: Optional[bool] = None,
```
Comment (erbridge) [inline replicate/prediction.py:615]: Why `block` and not `wait` or `sync`?

Comment (mattt) [reply replicate/prediction.py:615]: Because bike sheds.

Comment (erbridge) [reply replicate/prediction.py:615]: `block` sounds like a "bad" thing to me out of context. What am I blocking, etc?

Comment (mattt) [reply replicate/prediction.py]: Good news! With the move from `X-Sync: true` to `Prefer: wait`, this can now called `wait`.

**Pull request merged**